randovania/games/dread/json_data/Cataris.txt

```diff
@@ -4940,7 +4953,9 @@ Extra - asset_id: collision_camera_063
 > Right Ledge; Heals? False
   * Layers: default
   > Tunnel to Diffusion Beam Room (top)
-      Spider Magnet or Space Jump
+      Any of the following:
+          Grapple Beam or Use Spin Boost
+          Spider Magnet and Before Elun - Release X Parasites
```

Comment:
Grapple Beam needs either Spider or Grapple Movement. Simple IBJ can also be added as an option.

Reply:
this grapple movement makes sense to me and totally forgot the IBJ, was looking at the room solely for the spike in the wall not being present later